machine_management/adding-rhel-compute.adoc

```diff
@@ -25,6 +25,11 @@ include::modules/rhel-preparing-playbook-machine.adoc[leveloffset=+1]
 
 include::modules/rhel-preparing-node.adoc[leveloffset=+1]
 
+include::modules/rhel-attaching-instance-aws.adoc[leveloffset=+1]
+
+.Addtional Resources
+* See xref::../modules/installation-aws-permissions-iam-roles.adoc[Required AWS permissions for IAM roles]
```

Comment:
You have two `::` after the xref, so the link isn't working. Should only be one colon: `xref:`